keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -282,6 +291,11 @@ def tokenize(self, inputs):
             output_shape = tokens.shape.as_list()
             output_shape[-1] = self._sequence_length
             tokens = tokens.to_tensor(shape=output_shape)
+        # Convert to a dense output if input in scalar
+        if scalar_input:
+            tokens = tf.squeeze(tokens, 0)
+            tf.ensure_shape(tokens, shape=[None])
```

Comment:
Does the unit test below break without this? If it doesn't we might be able to remove.

If we do need it, we should probably update this set the shape to `[self._sequence_length]`. If sequence length is set, we do want to make sure the static shape is correct set to `[self._sequence_length]`.

Reply:
Yeah, it does fail without this check, I've made the changes to set shape appropriately now